app/src/main/res/values/styles.xml

```diff
@@ -108,6 +108,20 @@
         <item name="android:textStyle">normal</item>
     </style>
 
+    <style name="Header">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">@dimen/header_height</item>
+        <item name="android:background">@color/background</item>
+        <item name="android:gravity">center_vertical</item>
+        <item name="android:paddingLeft">16dp</item>
+        <item name="android:textSize">10sp</item>
+        <item name="android:fontFamily">sans-serif</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:textColor">@color/medium_grey</item>
+        <item name="android:letterSpacing">0.15</item>
+        <item name="android:lineSpacingExtra">2sp</item>
```

Comment:
This is probably what we discussed this morning. But why not use         
`<item name="android:textAllCaps">true</item>`here in the style and then have the Strings not necessarily all uppercase?

Reply:
I don't know if all headers are uppercase. Let's move this discussion out of this pr